### DISCUSSION TITLE
show better errors when web pages dont load

### DISCUSSION THREAD

**Discussion by shalva97** [minbrowser/min#1489] [Completed]
when something goes wrong, like on  this [link](https://developer.android.com/guide/components/activities), I get error codes that doesnot tell me much. At first glance nothing can be wrong with that link, it is developer.android.com, looks correct, it is also loaded in other tabs so I start blaming Min for it. Then I open Chromium and check if it also doesnot work there and it will tell me that it got ERR_TOO_MANY_REDIRECTS.

Can we have such error messages like Chromium? so I will know it is websites fault and not a bug in Min that needs to be investigated and reported...

**Comment by PalmerAL** [comment]
Done in https://github.com/minbrowser/min/commit/b3bc0592bba961839ca1c809af0c887ce95e5e58.
